prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -105,6 +111,21 @@ limits = {
   };
 }
 
+{{ if eq $GC_TYPE "generational" }}
```

Comment:
maybe add a comment here with the link to the prosody site for more information.

Reply:
added